src/ClusterBootstrap/template/storage/auto_share/setup_lustre.sh

```diff
@@ -0,0 +1,40 @@
+cat <<EOM >/etc/yum.repos.d/lustre.repo 
+[lustre-server]
+name=CentOS-$releasever - Lustre
+baseurl=https://downloads.hpdd.intel.com/public/lustre/latest-feature-release/el7/server/
+gpgcheck=0
+[e2fsprogs]
+name=CentOS-$releasever - Ldiskfs
+baseurl=https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/
+gpgcheck=0
+[lustre-client]
+name=CentOS-$releasever - Lustre
+baseurl=https://downloads.hpdd.intel.com/public/lustre/latest-feature-release/el7/client/
+gpgcheck=0
+EOM
+
+
+
+
+yum install -y e2fsprogs
+mkdir /tmp/lustre
+wget -O /tmp/lustre/kernel-3.10.0-1062.1.1.el7_lustre.x86_64.rpm https://downloads.whamcloud.com/public/lustre/lustre-2.13.0/el7.7.1908/server/RPMS/x86_64/kernel-3.10.0-1062.1.1.el7_lustre.x86_64.rpm
```

Comment:
Can you add detailed deployment instruction on which version of Azure VM to use for Lustre in README?